app/assets/javascripts/subscribers.js

```diff
@@ -0,0 +1,15 @@
+$(document).on("ajax:success", "#new_subscriber", function(e, data, status, xhr) {
```

Comment:
Line is too long.<br>'xhr' is defined but never used.<br>'status' is defined but never used.<br>'data' is defined but never used.<br>'e' is defined but never used.